tests/unit/kvmagent/test_mevoco_handlers.py

```
@@ -0,0 +1,836 @@
+
+
+@pytest.mark.kvmagent
+class TestMevocoBatchApplyUserdata:
+    def test_batch_apply_userdata_success(self):
+        plugin = _make_plugin()
+        _ensure_http()
+        plugin.userData_vms = {}
+
+        linux = cast(MagicMock, importlib.import_module("zstacklib.utils.linux"))
+        iproute = cast(MagicMock, importlib.import_module("zstacklib.utils.iproute"))
+        bash = cast(MagicMock, importlib.import_module("zstacklib.utils.bash"))
```

Ruff (0.15.2)

[error] Local variable `bash` is assigned to but never used

Remove assignment to unused variable `bash`

(F841)